src/fragmentarium/ui/images/CdliImages.test.tsx

```diff
@@ -21,8 +26,9 @@ beforeEach(async () => {
 })
 
 describe('CdliImages', () => {
-  beforeEach(() => {
+  beforeEach(async () => {
     renderImages()
+    await waitForElementToBeRemoved(() => screen.getByLabelText('Spinner'))
```

Comment:
Spinners are used a lot. There could be `waitForSpinners` function.

Reply:
@jlaasonen i would have to pass in screen and the function wouldn't reduce the code much can i just leave it for now ?

Reply:
It is not about reducing code. The same logic is duplicated in multiple places. If the spinner is changed all of them have to be updated. If there is a function to wait for the spinner only it has to be changed.

Reply:
alright then i will do it